src/utils.js

```diff
@@ -1,8 +1,13 @@
+var objectProto = Object.prototype;
+var hasOwnProperty = objectProto.hasOwnProperty;
+var toString = objectProto.toString;
+var symToStringTag = typeof Symbol != 'undefined' ? Symbol.toStringTag : undefined;
```

Comment:
`Symbol` may be undefined. but that is why the typeof check is there. this was lifted straight from `lodash` btw